src/Stateless/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyCopyright("Copyright © Stateless Contributors 2009-2016")]
 
 // This value will remain fixed throughout the 4.x series.
-[assembly: AssemblyVersion("4.0.0.0")]
+[assembly: AssemblyVersion("4.1.0.0")]
```

Comment:
This one shouldn't be updated, just the CSPROJ

Reply:
Ok, I'll sort it out.